helmfile/values/prometheus-alerts-sc.yaml.gotmpl

```diff
@@ -1,4 +1,7 @@
 esNodeCount: {{ add .Values.elasticsearch.dataNode.count .Values.elasticsearch.clientNode.count .Values.elasticsearch.masterNode.count }}
+eskubeAuditSizeGB: {{ .Values.elasticsearch.curator.retention.kubeAuditSizeGB }}
+eskubernetesSizeGB: {{ .Values.elasticsearch.curator.retention.kubernetesSizeGB }}
+esotherSizeGB: {{ .Values.elasticsearch.curator.retention.otherSizeGB }}
```

Comment:
Should you have one for `kubeaudit` as well?

Reply:
done